Automattic/wp-calypso
# Premium Themes: Redirect to Thank You page when activating a Premium theme

Resolves #73835

## Proposed Changes

- Redirect it to the Marketplace Thank You page when activating a Premium theme

## Testing Instructions


* Apply this branch in development which should have the feature flag `themes/display-thank-you-page` enabled
* Go to the themes page (`/themes/:site`)
* Select a premium theme and click on `Activate`
* You should be redirected to the Marketplace Thank You Page
* Disable the feature flag  (`/themes/:site?flags=-themes/display-thank-you-page `)
* Go to the themes page with the feature flag _disabled_: `/themes/:site?flags=-themes/display-thank-you-page `. Please note the `-` after the `flags=`
* Select a premium theme and click on `Activate`
* You should see the Thanks modal on the Themes page


## Pre-merge Checklist
